envi/pagelookup.py

```diff
@@ -66,5 +66,14 @@ def getMapLookup(self, va):
                 return marray[ va - mva ]
         return None
 
+    def delMapLookup(self, va):
+        for midx in range(len(self._maps_list)):
+            mva, mvamax, marray = self._maps_list[midx]
+            if va >= mva and va < mvamax:
+                return self._maps_list.pop(midx)
+
+        raise Exception('Address (0x%.8x) not in maps!' % va)
```

Comment:
`ValueError` is a bit more specific and appropriate here

Reply:
you could also make an argument for `IndexError` or `KeyError` i think

Reply:
hmmm...  indeed.